### DISCUSSION TITLE
Can't resolve 'react-query/types/react'

### DISCUSSION THREAD

**Discussion by theashishmaurya** [TanStack/query#4735] [Q&A]
Having this error while using react query v 3.39.2 

I don't understand whats causing this

![image](https://user-images.githubusercontent.com/50199029/210196968-22e101b0-c100-482c-8aac-e733249f3b50.png)

**Comment by Yalcingezsat** [comment]
An answer to this question has been given on AnswerDeveloper. hope it helps  

https://www.answerdeveloper.com/questions-answers/react-native-questions/having-this-error-while-using-react-query-v-3-39-2
